cas-server-support-jpa-ticket-registry/src/main/java/org/jasig/cas/ticket/registry/JpaTicketRegistry.java

```diff
@@ -168,8 +171,10 @@ public Ticket getTicket(final String ticketId) {
      */
     private Ticket getRawTicket(final String ticketId) {
         try {
-            if (ticketId.startsWith(this.ticketGrantingTicketPrefix)) {
-                return entityManager.find(TicketGrantingTicketImpl.class, ticketId);
+            for(final String prefix : this.ticketGrantingTicketPrefixes) {
```

Comment:
Checkstyle may be bug you about this. Please add a space between for and then "(".